react/finger-exercises/ecmascript/01 - spread/index.js

```diff
@@ -1,10 +1,36 @@
 import { isArray } from './utils';
 
 export function min() {
-  isArray();
-  return Math.min();
+  if(arguments.length === 0){
+    return undefined;
+  }
+
+  let numbers = arguments[0];
+  let args = Array.from(arguments);
+
+  if(isArray(numbers)){
+    return Math.min.apply(Math, numbers);
+  }
+  else {
+    return Math.min.apply(Math, args);
+  }
+
 }
 
-export function copy() {
+export function copy(a) {
+  if(isArray(a)){
+    return Object.assign([], a);
+  }
+  return Object.assign({}, a);
+}
 
+export function reverseMerge(a, b) {
+  return b.concat(a); 
 }
+
+export function filterAttribs() {
+  let mObject = copy(arguments[0]);
```

Comment:
You can use destructuring to get only the attributes you need, and then spread the remaining values using the spread operator.